### PR TITLE
`General`: Keep assessment editor in read only mode when answering a feedback request

### PR DESCRIPTION
**Pull request by Strohgelaender**
#### General
- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-tests/).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context
When answering a more feedback requests, it's not possible to modify the current assessment or add additional feedback items. They are not saved on the server. However, for feedback requests on automatically assessed programming exercises, the editor was still modifiable. This led to tutors adding feedback items and then referring to them in the answer text, causing students to not understand the incomplete answers. Some even tried to change the points during the feedback request.

### Description
On the student side, the explanation text telling that point changes are not possible is now no longer hidden behind a tooltip. This should make it more clear that changing the points is not possible.

When answering a feedback request, it's now not possible to add or edit feedback.

### Steps for Testing
Prerequisites:
- 1 Tutor
- 1 Student
- 1 Programming Exercise with automatic Complaints enabled

1. Submit a Feedback Request as a Student
2. Answer the request as a tutor. Verify that editing feedback is no longer possible.

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Test Coverage
unchanged. Added a test that catches this case (fails when used against develop).

### Screenshots
old:
grafik

new:
grafik

### CODE REVIEW

File: src/main/webapp/app/complaints/form/complaints-form.component.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div class="col-12 mt-4" *ngIf="isCurrentUserSubmissionAuthor">
     <div class="row">
-        <div class="col-12">
-            <h3>{{ complaintType === ComplaintType.COMPLAINT ? ('artemisApp.complaint.title' | artemisTranslate) : ('artemisApp.moreFeedback.title' | artemisTranslate) }}</h3>
+        <div class="col-12 col-md-6">
+            <h3>{{ 'artemisApp.' + (complaintType === ComplaintType.COMPLAINT ? 'complaint' : 'moreFeedback') + '.title' | artemisTranslate }}</h3>
```
Comment (julian-christl) [inline src/main/webapp/app/complaints/form/complaints-form.component.html:4]: I would revert this. If someone searches for occurrences, they won't find this.

**Comment by julian-christl** [comment]
@Strohgelaender Exam Mode Test is not necessary, in my opinion. The functionality does not relate to it in any way.

File: src/main/webapp/app/complaints/form/complaints-form.component.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <div class="col-12 mt-4" *ngIf="isCurrentUserSubmissionAuthor">
     <div class="row">
-        <div class="col-12">
+        <div class="col-12 col-md-6">
             <h3>{{ complaintType === ComplaintType.COMPLAINT ? ('artemisApp.complaint.title' | artemisTranslate) : ('artemisApp.moreFeedback.title' | artemisTranslate) }}</h3>
             <p *ngIf="complaintType !== ComplaintType.COMPLAINT">
                 {{ 'artemisApp.moreFeedback.' + (exercise.teamMode ? 'descriptionTeam' : 'description') | artemisTranslate }}
```
Comment (julian-christl) [inline src/main/webapp/app/complaints/form/complaints-form.component.html:6]: When we're already on it, would you mind?
```suggestion
                {{ exercise.teamMode ? 'artemisApp.moreFeedback.descriptionTeam' : 'artemisApp.moreFeedback.description' | artemisTranslate }}
```

**Review by Vinicius-Agreste — APPROVED** [review]
Tested on ts2. Working as expected.

**Review by julian-christl — APPROVED** [review]
cr

**Review by simon-hng — APPROVED** [review]
Code

**Review by tobias-lippert — CHANGES_REQUESTED** [review]
Tested on ts1.  I was still able to add feedback.
another thing I noticed, but might be something for another PR is that when submitting a feedback request as a student the text field only changes to readonly after reloading the page but not after clicking request more feedback.

**Review by tobias-lippert — APPROVED** [review]
Tested on ts1. Works as expected.

**Review by YeetTheFirst21 — APPROVED** [review]
Tested on TS2, seems to be working as intended

Screenshot 2023-03-02 111106

**Pull request merged**